### PR TITLE
Added javadocs to CastStrings.fromIntegersWithBase method [skip ci]

### LINKED ISSUE #1451: [DOC] Add javadoc headers to `fromIntegersWithBase` method in CastStrings.java
## Report incorrect documentation

**Location of incorrect documentation**
No documentation exists currently. 

**Describe the problems or issues found in the documentation**
NA

**Steps taken to verify documentation is incorrect**
NA

**Suggested fix for documentation**
Include Java docs

### PR DESCRIPTION
**Pull request by razajafri**
This PR just adds some javadocs to the method 

fixes #1451

### CODE REVIEW

File: src/main/java/com/nvidia/spark/rapids/jni/CastStrings.java [hunk 1]
```diff
@@ -109,6 +109,21 @@ public static ColumnVector toIntegersWithBase(ColumnView cv, int base,
       type.getTypeId().getNativeId()));
   }
 
+  /**
+   * Converts an integer column to a strings column with either decimal or hexadecimal values 
+   * based on the base provided. The hexadecimal value will be returned with the leading
+   * zeros dropped
```
Comment (gerashegalov) [inline src/main/java/com/nvidia/spark/rapids/jni/CastStrings.java:115]: Again maybe in a "Note:" say that there are no leading zeros or padding of digits to a particular bit width

File: src/main/java/com/nvidia/spark/rapids/jni/CastStrings.java [hunk 2]
```diff
@@ -109,6 +109,21 @@ public static ColumnVector toIntegersWithBase(ColumnView cv, int base,
       type.getTypeId().getNativeId()));
   }
 
+  /**
+   * Converts an integer column to a strings column with either decimal or hexadecimal values
```
[the remark this answers is not in the document]
Comment (razajafri) [reply src/main/java/com/nvidia/spark/rapids/jni/CastStrings.java:113]: Right now we only support 2 bases and it's ok to have it in the main description. We can make it more generic when time comes and we support more bases

**Comment by razajafri** [comment]
build

File: src/main/java/com/nvidia/spark/rapids/jni/CastStrings.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (razajafri) [reply src/main/java/com/nvidia/spark/rapids/jni/CastStrings.java:113]: Thank you for taking time out to review this. If you still feel strongly I can make the change. PTAL

**Comment by razajafri** [comment]
@gerashegalov I have updated the PR based on our discussion offline

**Comment by razajafri** [comment]
build

File: src/main/java/com/nvidia/spark/rapids/jni/CastStrings.java [hunk 3]
```diff
@@ -109,6 +109,24 @@ public static ColumnVector toIntegersWithBase(ColumnView cv, int base,
...
+   * Converts an integer column to a string column by converting the underlying integers to the
+   * specified base.
+   *
+   * Note: Right now we only support base 10 and 16. The hexadecimal values will be
+   * returned without leading zeros or padding at the end
+   * 
+   * Example:
+   * input = [123, -1, 0, 27, 342718233]
+   * s = fromIntegersWithBase(input, 16)
+   * s is [ '4D2', 'FFFFFFFF', '0', '1B', '146D7719']
```
Comment (gerashegalov) [inline src/main/java/com/nvidia/spark/rapids/jni/CastStrings.java:122]: is this accurate? 123 should map to "7B"

**Review by gerashegalov — APPROVED** [review]
LGTM

File: src/main/java/com/nvidia/spark/rapids/jni/CastStrings.java (hunk 3 above)
Comment (razajafri) [reply src/main/java/com/nvidia/spark/rapids/jni/CastStrings.java]: Yes, you are right. I copied this from cudf doc [here](https://github.com/rapidsai/cudf/blob/23d24d43fac8615166c38231f13fe8751a8aec42/cpp/include/cudf/strings/convert/convert_integers.hpp#L204). 

I have created a ticket in cudf https://github.com/rapidsai/cudf/issues/14232

**Comment by razajafri** [comment]
build

**Comment by razajafri** [comment]
build

**Comment by razajafri** [comment]
build

**Pull request merged**